src/collections/_documentation/accounts/quotas/manage-event-stream-guide.md

```diff
@@ -3,30 +3,30 @@ title: Manage Your Event Stream, a Guide
 sidebar_order: 3
 ---
 
-Getting started with Sentry error monitoring is pretty straightforward --- create a Sentry project, install and initialize the Sentry SDK in your code and you're ready to go. Once completed, **any** Exception, Runtime Error or Crash that occurs **wherever** and **whenever** your code is running, will be aggregated in your `Event Stream`. Depending on your portfolio, your stream might be aggregating millions of events in no time. Sending all your errors to Sentry is generally a good practice. This ensures greater visibility into the health and performance of your application code. However, with just this basic setup in place, you might risk:
+Sending all of your application errors and crashes to Sentry ensures that you'll be notified in real-time as errors happen in your code. However, with just the basic setup in place, you might soon realize that your applications are generating a lot of errors, and those might trigger quite a few notifications. Sentry provides various tools that give you control over the _type_ and _amount_ of errors that are monitored. These will allow you to:
 
-1. Flooding your event stream with excessive noise such as errors from 3rd party libraries, errors from unstable versions of your code, etc.
-2. Getting spammed with alert notifications about things you don't care about or worse --- lose track of the critical issues that are hiding in your code.
+1. Shape your event stream to make it actionable and meaningful.
+2. Reserve those real-time notifications for errors that actually break your code.
+3. If you're on an event-quota based plan, unnecessary noise might end up draining your quota and leaving you blind when and where it matters.
```

Comment:
Suggest: "Manage your quotas if you're on an event-quota based plan, since too much noise may drain your quota."